graphql_compiler/tests/test_helpers.py

```diff
@@ -407,3 +435,13 @@ def get_test_macro_registry():
     for graphql, args in valid_macros:
         register_macro_edge(macro_registry, graphql, args)
     return macro_registry
+
+
+def ordered(obj):
+    """Return an ordered representation of a json-serializable object"""
+    if isinstance(obj, dict):
+        return OrderedDict(sorted([(k, ordered(v)) for k, v in obj.items()], key=str))
+    if isinstance(obj, list):
+        return sorted([ordered(x) for x in obj], key=str)
```

Comment:
This is probably fine but somewhat risky. `str` _should_ be deterministic, but it is definitely getting called multiple times and so if it ever isn't deterministic, we're going to have a really rough time figuring out what went wrong.